.github/workflows/release.yml

```
@@ -0,0 +1,258 @@
+            fi
+          fi
+
+          echo "Bump type: $BUMP_TYPE"
+
+          # Handle staging releases with pre-release tags
+          if [ "$IS_STAGING" = "true" ]; then
+            # For staging, create a pre-release version
+            # Get current version and add alpha pre-release
+            CURRENT_VERSION=$(node -p "require('./package.json').version")
+            echo "Current version: $CURRENT_VERSION"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)